public/locations-list-map.js

```diff
@@ -148,6 +150,23 @@ const addDonationSites = () => {
   }
 };
 
+const translatedFilterItems = () => {
+  const translated = {};
+  const filterItemKeys = Object.keys(FILTER_ITEMS);
+
+  for (let i = 0; i < filterItemKeys.length; i++) {
+    const filterItemKey = filterItemKeys[i];
+    const filterItem = FILTER_ITEMS[filterItemKey];
+
+    translated[filterItemKey] = {
+      name: $.i18n(filterItem.name),
```

Comment:
needed to wait until i18n has loaded before calling which is why these can't live in the other file

Reply:
This should be a code comment...